### PR TITLE
Update links to grant programs

### PR DESCRIPTION
**Pull request by nebolax**
Some links to grant programs became outdated and in this PR I fixed it.

### CODE REVIEW

File: src/content/community/grants/index.md [hunk 1]
```diff
@@ -28,17 +28,16 @@ These programs support the broad Ethereum ecosystem by offering grants to a wide
 These projects have created their own grants for projects aimed at developing and experimenting with their own technology.
 
 - [Aave Grants Program](https://aavegrants.org/) – _[Aave](https://aave.com/) grants DAO_
-- [Balancer](https://balancergrants.notion.site/Balancer-Community-Grants-23e562c5bc4347cd8304637bff0058e6) – _[Balancer](https://balancer.fi/) ecosystem fund_
+- [Balancer](https://quark-ceres-740.notion.site/Balancer-Grants-938f1b979810427f8d903a904315da41) – _[Balancer](https://balancer.fi/) ecosystem fund_
 - [Chainlink Grants Program](https://chain.link/community/grants) - _[Chainlink](https://chain.link/) community grants_
-- [Compound Grants Program](https://compoundgrants.org/) – _[Compound](https://compound.finance/) finance ecosystem_
+- [Compound Grants Program](https://twitter.com/compoundgrants/status/1615562418297659393?s=20) – _[Compound](https://compound.finance/) finance ecosystem_
```
Comment (minimalsm) [inline src/content/community/grants/index.md:33]: I'm not really happy about linking to a Twitter post here; is there a better way?

File: src/content/community/grants/index.md [hunk 2]
```diff
@@ -28,17 +28,16 @@ These programs support the broad Ethereum ecosystem by offering grants to a wide
 These projects have created their own grants for projects aimed at developing and experimenting with their own technology.
 
 - [Aave Grants Program](https://aavegrants.org/) – _[Aave](https://aave.com/) grants DAO_
-- [Balancer](https://balancergrants.notion.site/Balancer-Community-Grants-23e562c5bc4347cd8304637bff0058e6) – _[Balancer](https://balancer.fi/) ecosystem fund_
+- [Balancer](https://quark-ceres-740.notion.site/Balancer-Grants-938f1b979810427f8d903a904315da41) – _[Balancer](https://balancer.fi/) ecosystem fund_
```
[the remark this answers is not in the document]
Comment (nebolax) [reply src/content/community/grants/index.md:31]: Hmm this link does not open for me. Does it work fine for you or did you make a mistake?

File: src/content/community/grants/index.md (hunk 1 above)
Comment (nebolax) [reply src/content/community/grants/index.md:33]: The problem is that there are multiple grant programs and I have not found another link / page that would contain all of them.

I can use https://linktr.ee/ or some similar service and just paste the links there. Will it be better?

**Comment by nebolax** [comment]
Hi, @minimalsm! Any update on my replies?

File: src/content/community/grants/index.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (minimalsm) [reply src/content/community/grants/index.md:31]: Sorry, totally missed this reply. https://grants.balancer.community/ is a forwarding link, what do you see?

File: src/content/community/grants/index.md (hunk 1 above)
Comment (minimalsm) [reply src/content/community/grants/index.md:33]: So looks like they use Questbook to manage this, and post links to new grants rounds on Twitter. I don't think we can maintain this, so I would just remove it entirely.

File: public/content/community/grants/index.md [hunk 3]
```diff
@@ -28,17 +28,16 @@ These programs support the broad Ethereum ecosystem by offering grants to a wide
 These projects have created their own grants for projects aimed at developing and experimenting with their own technology.
 
 - [Aave Grants Program](https://aavegrants.org/) – _[Aave](https://aave.com/) grants DAO_
-- [Balancer](https://balancergrants.notion.site/Balancer-Community-Grants-23e562c5bc4347cd8304637bff0058e6) – _[Balancer](https://balancer.fi/) ecosystem fund_
+- [Balancer](https://quark-ceres-740.notion.site/Balancer-Grants-938f1b979810427f8d903a904315da41) – _[Balancer](https://balancer.fi/) ecosystem fund_
```
Comment (minimalsm) [inline public/content/community/grants/index.md:31]:
```suggestion
- [Balancer](https://grants.balancer.community/) – _[Balancer](https://balancer.fi/) ecosystem fund_
```

**Review by minimalsm — APPROVED** [review]
Added some changes, LGTM now :)

**Comment by minimalsm** [comment]
Thanks again!

@all-contributors please add @nebolax for docs

**Pull request merged**